src/requestLogging/requestLogging.ts

```diff
@@ -66,6 +68,11 @@ const replaceHeaders = (
   return cleanedHeaders;
 };
 
+/**
+ * Returns context fields for the passed Koa context
+ */
+export type ContextFields = (ctx: Koa.Context) => Record<string, unknown>;
```

Comment:
Would it be worth aligning this closer with `RequestLogging.createMiddleware`:

```suggestion
export type ContextFields = (ctx: Koa.Context, fields: Fields) => Record<string, unknown>;
```

This would make it easier to extend the context without having to manually invoke the baseline `contextFields`:

```typescript
createLoggerContextStorage((ctx, fields) => ({
  ...fields,
  additionalField: getAdditionalField(ctx),
}));
```

Reply:
Oooo yep that would make sense